### ISSUE TITLE
iOS app opens with no notes and takes a long time to show my notes

### ISSUE THREAD

**Issue by gingerbeardman** [glushchenko/fsnotes#1777]
### Description

I noticed this fairly recently, but not sure if it is expected behaviour or a bug or something specific to me.

iOS app opens with no notes and takes a long time, several seconds after app launch for my notes to appear.

Total time to being able to edit is ~13 seconds

### To Reproduce

1. open app
2. wait through logo
3. wait through "loading"
4. wait through nothingness
5. finally—my notes!

### Expected behavior

- quick!

### FSNotes version

- 6.9.4 (308)

### macOS/iOS version

- 17.7

### Additional context

Here's a screen recording:

https://github.com/user-attachments/assets/30756cf7-e785-41b3-b443-14bff05166fa

**Comment by gingerbeardman** [comment]
I did a clean up and removed some really large research filename listings and it's super quick now.

So, it seems that it must go through every file on startup? Seems wasteful?

**Comment by glushchenko** [comment]
I put in four copies of "war and peace", and I'm having a very long load time too. 


I'm working on it now.

**Comment by gingerbeardman** [comment]
that'll do it LOL

**Comment by GitterGuester** [comment]
I have experienced this exact issue as a fairly new user and with many-but-very-small notes imported from my old NV notes, so I’m not sure it’s a size thing.  Unfortunately I experience it intermittently with no discernible pattern (ie mobile data, location, signal strength). Often it just eventually crashes and I re-open and it works or sometimes not and I just have to abandon the task.  I posted about it on the Reddit feed as I’m not a geeky gal or a software person so am baffled and intimidated by this platform! https://www.reddit.com/r/FSNotes/comments/1f6sd0x/fs_notes_fail_on_ios_phone_w_mobile_data_how_to/
